core-api/src/main/java/com/optimizely/ab/decision/experiment/CompositeExperimentDecisionService.java

```diff
@@ -0,0 +1,66 @@
+/****************************************************************************
+ * Copyright 2019, Optimizely, Inc. and contributors                        *
+ *                                                                          *
+ * Licensed under the Apache License, Version 2.0 (the "License");          *
+ * you may not use this file except in compliance with the License.         *
+ * You may obtain a copy of the License at                                  *
+ *                                                                          *
+ *    http://www.apache.org/licenses/LICENSE-2.0                            *
+ *                                                                          *
+ * Unless required by applicable law or agreed to in writing, software      *
+ * distributed under the License is distributed on an "AS IS" BASIS,        *
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. *
+ * See the License for the specific language governing permissions and      *
+ * limitations under the License.                                           *
+ ***************************************************************************/
+package com.optimizely.ab.decision.experiment;
+
+import com.optimizely.ab.config.Experiment;
+import com.optimizely.ab.decision.entities.DecisionStatus;
+import com.optimizely.ab.decision.entities.ExperimentDecision;
+import com.optimizely.ab.decision.entities.Reason;
+import com.optimizely.ab.event.internal.UserContext;
+import com.optimizely.ab.internal.ExperimentUtils;
+import com.optimizely.ab.decision.experiment.service.ExperimentBucketerDecisionService;
+
+import java.util.List;
+import java.util.Arrays;
+
+import javax.annotation.Nonnull;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * ExperimentDecisionContext contains the information needed to be able to make a decision for a given experiment
+ */
+public class CompositeExperimentDecisionService implements IExperimentDecisionService {
+
+    private static final Logger logger = LoggerFactory.getLogger(CompositeExperimentDecisionService.class);
+    /**
+     * @return List of Decision service for experiment
+     */
+    private List<IExperimentDecisionService> getExperimentServices() {
+        return Arrays.asList(
+            new ExperimentBucketerDecisionService()
```

Comment:
Can you please add TODO, I believe more Decision services to be added. Please add TODO comment.